redash/query_runner/oracle.py

```diff
@@ -130,7 +130,11 @@ def output_handler(cls, cursor, name, default_type, length, precision, scale):
                 )
 
     def run_query(self, query, user):
-        connection = cx_Oracle.connect(self.connection_string)
+        dsn_t = cx_Oracle.makedsn(
+            self.configuration["host"],
+            self.configuration["port"],
+            service_name=self.configuration["servicename"])
+        connection = cx_Oracle.connect(user=self.configuration["user"],password=self.configuration["password"], dsn=dsn_t)
```

Comment:
Thanks, but this is make the creation of `connection_string` in the constructor redundant. Let's remove the dead code there as well.

Reply:
Just updated it, thanks